### PR TITLE
WIP: Containerci

### PR DESCRIPTION
**Pull request by joejstuart**

### CODE REVIEW

**Review by arilivigni — APPROVED** [review]
LGTM

**Review by dirgim — APPROVED** [review]
LGTM, nice work

File: grafana/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+RUN yum -y install git unzip nss_wrapper && \
+    curl -L -o /tmp/grafana.rpm https://s3-us-west-2.amazonaws.com/grafana-releases/release/${GRAFANA_RPM} && \
+    yum -y localinstall /tmp/grafana.rpm && \
+    yum -y clean all && \
+    rm -rf /var/cache/yum && \
+    rm /tmp/grafana.rpm
+
+COPY ./s2i/bin /usr/libexec/s2i
+COPY ./usr/bin/ /usr/bin/
+COPY ./usr/share/grafana/ /usr/share/grafana
```
Comment (dirgim) [inline grafana/Dockerfile:22]: Just noticed this dir is missing.

File: jenkins/slave/test/run [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  if [[ "$result" != "0" ]]; then
+    echo "S2I image '${IMAGE_NAME}' test FAILED (exit code: ${result})"
+    cleanup
+    exit $result
+  fi
+}
+
+trap cleanup EXIT SIGHUP SIGINT SIGTERM
+
+prepare
```
Comment (dirgim) [inline jenkins/slave/test/run:52]: This function is missing here, but I'm guessing it's not needed anyway?

File: grafana/Dockerfile (hunk 1 above)
Comment (joejstuart) [reply grafana/Dockerfile:22]: Forgot to add it. thank you

File: jenkins/slave/test/run (hunk 2 above)
Comment (joejstuart) [reply jenkins/slave/test/run:52]: I need to fix that. thanks

**Review by arilivigni — APPROVED** [review]
LGTM

**Pull request merged**